infinitest-runner/src/main/java/org/infinitest/testrunner/junit5/Junit5Runner.java

```diff
@@ -27,24 +27,21 @@
  */
 package org.infinitest.testrunner.junit5;
 
-import static org.junit.platform.engine.discovery.DiscoverySelectors.selectClass;
-
-import java.util.ArrayList;
-import java.util.List;
-
 import org.infinitest.config.InfinitestConfigurationSource;
 import org.infinitest.testrunner.TestResults;
-import org.junit.jupiter.engine.JupiterTestEngine;
 import org.junit.platform.engine.Filter;
-import org.junit.platform.launcher.EngineFilter;
 import org.junit.platform.launcher.Launcher;
 import org.junit.platform.launcher.LauncherDiscoveryRequest;
 import org.junit.platform.launcher.TagFilter;
-import org.junit.platform.launcher.TestExecutionListener;
 import org.junit.platform.launcher.TestPlan;
 import org.junit.platform.launcher.core.LauncherDiscoveryRequestBuilder;
 import org.junit.platform.launcher.core.LauncherFactory;
 
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.platform.engine.discovery.DiscoverySelectors.selectClass;
+
```

Comment:
I wonder if `Junit5Runner` shouldn't be renamed to `JUnit5PlatformRunner` or something similar after these changes.